java/ql/lib/semmle/code/java/frameworks/android/Intent.qll

```diff
@@ -77,25 +63,68 @@ class AndroidServiceIntentMethod extends Method {
 
 /**
  * The method `Context.startActivity` or `startActivities`.
+ *
+ * DEPRECATED: Use `StartActivityMethod` instead.
```

Comment:
We should probably create an additional `deprecated` change note mentioning this.